### ISSUE TITLE
Clicking on variable technical name record not found

### ISSUE THREAD

**Issue by markdangio** [arnoudkooi/ServiceNow-Utils#439]
When /tn is active and you are in the platform native view of a sc_req_item record and you click on the hyperlinked variable technical name, the link is broken. It opens a new tab to /question.do?sys_id=……. It should be /sc_item_option.do?sys_id=……

**Comment by arnoudkooi** [comment]
Not sure why I made below change

https://github.com/arnoudkooi/ServiceNow-Utils/blob/82271d79cc8a90b50b95c050ece20322706a01d0/inject.js#L2587

I'll revert it to

```javascript 
var linkTableName = tableName == 'ni' ? 'item_option_new' : 'sc_item_option';
```

**Comment by markdangio** [comment]
Thank you!On Oct 4, 2023, at 1:35 PM, arnoudkooi ***@***.***> wrote:﻿
Not sure why I made below change
https://github.com/arnoudkooi/ServiceNow-Utils/blob/82271d79cc8a90b50b95c050ece20322706a01d0/inject.js#L2587
I'll revert it to
var linkTableName = tableName == 'ni' ? 'item_option_new' : 'sc_item_option';

**Comment by arnoudkooi** [comment]
In UI16? I think you then should disable this setting (be sure to refresh the page for setting to take affect.)

![Google Chrome20231004 - 1958](https://github.com/arnoudkooi/ServiceNow-Utils/assets/840781/bf519721-990f-43fe-93d3-0ed911d1f9e4)

**Comment by arnoudkooi** [comment]
This should be fixed in 7.1.5.0, please let me know if it works again.

**Comment by arnoudkooi** [comment]
Assume its ok.

**Comment by markdangio** [comment]
Yes all good! Thanks!On Oct 13, 2023, at 5:02 PM, arnoudkooi ***@***.***> wrote:﻿
Closed #439 as completed.
